chia/plotting/util.py

```diff
@@ -16,8 +16,8 @@
 @dataclass
 class PlotsRefreshParameter:
     interval_seconds: int = 120
-    batch_size: int = 30
-    batch_sleep_milliseconds: int = 10
+    batch_size: int = 300
+    batch_sleep_milliseconds: int = 5
```

Comment:
I bet you can even make this 1ms. The only reason we need this sleep is to allow other already-queued-up co-routines a chance to run. We'll be put at the end of the run-queue no matter what the sleep time is, I would expect

Reply:
Tested and seems to still have the same behavior :+1:

Reply:
,!!!!
- [ ] ~~~~